src/components/CreateVmWizard/steps/SummaryReview.js

```diff
@@ -59,7 +59,14 @@ const ReviewBasic = ({ id, dataCenters, clusters, isos, templates, operatingSyst
     <Item id={`${id}-memory`} label={msg.memory()}>{userFormatOfBytes(basic.memory, 'MiB').str}</Item>
     <Item id={`${id}-cpus`} label={msg.cpus()}>
       { basic.cpus }
-      {/* TODO: Include topology (as a popover?) */}
+      <FieldLevelHelp
+        content={msg.totalCpuTooltip({
+          cores: basic.topology.cores,
+          sockets: basic.topology.sockets,
+          threads: basic.topology.threads,
+        })}
+        inline
+      />
```

Comment:
It's not must, but I have issue #1094 which deal with tooltips so I'll suggest you to use `<InfoCircleIcon />` wrapped by `<OverlayTooltip>`